static/js/components/EducationTab.js

```diff
@@ -37,6 +40,15 @@ class EducationTab extends ProfileTab {
     this.handleCloseDialog = this.handleCloseDialog.bind(this);
 
   }
+
+  componentDidMount() {
+    // Use dialog polyfill to allow dialog usage on browsers other than Chrome and Opera (as of early 2016)
+    const node = ReactDOM.findDOMNode(this);
+    for (let dialog of node.querySelectorAll("dialog")) {
+      dialogPolyfill.registerDialog(dialog);
+    }
+  }
```

Comment:
would it make more sense to do this in `App.js`? Then we can use dialogs anywhere across the dashboard.